packages/hint/src/lib/types/html.ts

```diff
@@ -0,0 +1,131 @@
+import * as parse5 from 'parse5';
+import * as htmlparser2Adapter from 'parse5-htmlparser2-tree-adapter';
+import * as cssSelect from 'css-select';
+
+import {
+    INamedNodeMap,
+    ProblemLocation
+} from '../types';
+
+type Attrib = {
+    [key: string]: string;
+};
+
+export type ParsedHTMLElement = Element & {
```

Comment:
> Does a ParsedHTMLElement actually include support for everything Element defines (since this seems to refer to the built-in Element type)?

Nop. It support just some `element` properties.